src/lib/routes/client-api/feature.ts

```diff
@@ -199,6 +227,10 @@ export default class FeatureController extends Controller {
         req: IAuthRequest,
         res: Response<ClientFeaturesSchema>,
     ): Promise<void> {
+        if (this.flagResolver.isEnabled('optimal304')) {
+            return this.optimal304(req, res);
+        }
```

Comment:
Maybe we can shadow traffic to this branch and compare to detect if there's a different behavior using one method or the other. We can do it in sandbox at expense of some additional latency

Reply:
We can even compare time/size

Reply:
Oh, in that case the new behavior should be available on a separate path also.

Reply:
I added a separate toggle called "optimal304Differ". It will allow us to perform actual diffing of the way of caching feature toggles (which end up with a db query every 600ms) and the new way of cache feature toggles, by using revision id as part of cache key and cache for 1 hour. Thus would allow us to validate the new cache implementation behind the scene for real customers.